src/x86/ssse3.rs

```diff
@@ -8,10 +8,45 @@ pub fn _mm_abs_epi8(a: i8x16) -> u8x16 {
     unsafe { pabsb128(a) }
 }
 
+
+#[inline(always)]
+#[target_feature = "+ssse3"]
```

Comment:
Please follow the style in the rest of the code. :-) Put the attributes after the comments.